zerver/openapi/zulip.yaml

```diff
@@ -3210,6 +3211,19 @@ paths:
                                       description: |
                                         The new description of the group. Only present if the description
                                         changed.
+                                    can_join_group:
+                                      allOf:
+                                        - $ref: "#/components/schemas/GroupSettingValue"
+                                        - description: |
+                                            A [group-setting value][setting-values] defining the set of users who
+                                            have permission to join this group. Only present if this user group
+                                            permissiod setting changed.
+
+                                            **Changes**: New in Zulip 10.0 (feature level 300).
+
+                                            Will be one of the following:
```

Comment:
This seems oddly cut off? In any case, I don't think we want to be documenting values here; we should instead be referring to common documentation on how a client can see what values are possible dynamically.

Maybe I'll do a preparatory PR for that.

Reply:
Opened https://github.com/zulip/zulip/pull/31910 to clean this detail of our framework up. @shubham-padia @Vector73 FYI, since you'll want to avoid adding more copies of the pattern I'm deleting there.

Reply:
This is shown as this -
![Screenshot from 2024-10-08 16-49-13](https://github.com/user-attachments/assets/38decaf2-8338-45f4-9dea-3944d489f52d)